development/webpack/utils/cli.ts

```diff
@@ -374,6 +376,20 @@ function getOptions(
       type: 'boolean',
     },
 
+    metamaskEnvironment: {
+      array: false,
+      choices: environmentOptions,
+      default: undefined as string | undefined,
```

Comment:
Why is a type assertion needed here?
Given that the default is `undefined`, could we omit the default instead?